dace/sdfg/nodes.py

```diff
@@ -925,6 +929,8 @@ class Map(object):
                                  optional=True,
                                  optional_condition=lambda m: m.schedule in dtypes.GPU_SCHEDULES)
 
+    gpu_force_syncthreads = Property(dtype=bool, desc="Force a call to the __syncthreads for the map", default=False)
+
```

Comment:
Unrelated

Reply:
Must have sneaked in from the other PR, will delete it